src/python/bot/untrusted_runner/host.py

```diff
@@ -157,10 +156,7 @@ def wrapped(*args, **kwargs):
         # For timeouts, which aren't fatal errors, resurface the right
         # exception.
         if 'TimeoutError' in repr(e):
-          # TODO(ochang): Replace with generic TimeoutError in Python 3.
-          # Use __str__ to avoid newstrs. TODO(ochang): Use plain str() once
-          # migrated to Python 3.
-          raise engine.TimeoutError(e.__str__())
+          raise TimeoutError(e.str())
```

Comment:
did you mean str(e) ? @oliverchang to review.

Reply:
This is left for review.

Reply:
I think it is str(e), lets do and then @oliverchang can do final review and land.